Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:22.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+ENV TZ=Etc/UTC
+
+RUN apt-get update && apt-get install -y \
+    apt-transport-https \
+    curl \
+    git \
+    locales \
+    mysql-client \
+    nginx \
+    software-properties-common \
+    unzip \
+    tzdata \
+    vim
+
+# System-level PHP setup
+RUN add-apt-repository ppa:ondrej/php -y
+RUN apt-get install -y \
+    php7.4 php7.4-curl php7.4-fpm php7.4-gd php7.4-mbstring php7.4-mysql php7.4-xml
+RUN mkdir /run/php # For the php-fpm socket file
```

Comment:
`/var/run/php/` is used in the nginx config below...

And does this actually need doing manually?